src/CallToAction.elm

```diff
@@ -18,8 +18,11 @@ callToActionButton callToAction aClass =
                 [ getIcon callToAction.icon (Just "button--icon")
                 , span [] [ text callToAction.promptLong ]
```

Comment:
@Nikomus does it need to be in different spans for styling?
Or could go with a `, span [] [ text (callToAction.promptLong ++ " ") ]`